src/FSharpVSPowerTools/FSharpVSPowerTools.csproj

```diff
@@ -46,7 +46,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DeployExtension>True</DeployExtension>
+    <DeployExtension>False</DeployExtension>
```

Comment:
Two reasons:
* Having this on makes building in VS 2015 much slower. No idea why.
* Exp VS 2015 instance does not work anyway (I mean absolutely, never works), so it does not make any sense to deploy VSIX to it during each building.